server.py

```diff
@@ -5,430 +5,457 @@
 from flask import Flask, request, send_from_directory
 
 
-app = Flask(__name__, static_url_path = "")
+app = Flask(__name__, static_url_path="")
 
 # Load configuration
 app.config.from_pyfile("config-example.py")  # Use example for defaults
 if os.path.isfile(os.path.join(app.root_path, "config.py")):
-        app.config.from_pyfile("config.py")
+    app.config.from_pyfile("config.py")
 
 
 # Views
 
+
 @app.route("/")
 def index():
-	return app.send_static_file("index.html")
+    return app.send_static_file("index.html")
```

Comment:
no

Reply:
PEP-8 recommends using four spaces, so in the interest of following established standards, we ought to do that :smiley:

Reply:
I know that PEP-8 recommends this, but I don't like it and never will.
It's still useful to be close to the standard, so I'd just disable this behaviour in the formatter.

Reply:
[Black doesn't support tabs](https://github.com/ambv/black/issues/47), so I'm afraid that's not an optino.